spaceship/lib/spaceship/du/utilities.rb

```diff
@@ -16,6 +16,8 @@ def content_type(path)
       return 'video/quicktime' if path.end_with?('.mov')
       return 'video/mp4' if path.end_with?('.m4v')
       return 'video/mp4' if path.end_with?('.mp4')
+      return 'text/plain' if path.end_with?('.txt')
+      return 'application/zip' if path.end_with?('.zip')
```

Comment:
What do you think about adding `application/pdf`? (per OP request)
.zip is a good catch-all, though.